app/assets/javascripts/components/plant_population_submission.coffee

```diff
@@ -173,6 +173,11 @@ window.PlantPopulationSubmission = class PlantPopulationSubmission extends Submi
             $li = $("<li></li>").text(error)
             $errors.find("ul").append($li)
           )
+        else
+          @$('.fileinput-button').removeClass('disabled')
+
+          $errors = $(".errors").text("").removeClass('hidden').append("<ul></ul>")
+          $errors.find("ul").append($("<li></li>").text("Unexpected server response: #{data.jqXHR.status} #{data.jqXHR.statusText}"))
```

Comment:
Do you remember what kind of file you uploaded? I only managed to get this 500 response using a badly encoded file, otherwise I get regular error message such as "File content type is invalid".

Reply:
I did it with two files, one was even a CSV, only a completely different one (than expected). I also remember I selected nothing on the 2nd step. The 3rd step was also empty when I uploaded the file.

Reply:
OK, fixed.